src/js/themes/base.js

```diff
@@ -1119,11 +1119,17 @@ export const generate = (baseSpacing = 24, scale = 6) => {
         },
         width: '2px',
       },
+      background: {
+        color: {
+          dark: 'light-4',
+          light: 'light-1',
+        },
+      },
```

Comment:
I'm thinking we could not set defaults for the background color and just leave it as
```
// background: {
//    color: undefined,
//  },
```